docs/topics/plugin-identify-resolve-issues.adoc

```diff
@@ -3,17 +3,54 @@
 // * docs/plugin-guide/master.adoc
 
 [id='plugin-identify-resolve-issues_{context}']
-= Troubleshooting migration issues
+= Using the {PluginName}
```

Comment:
I think this title could be more precise. Something like "Analyzing your projects with the MTA IDE plugin" or similar. "Using" is too vague.

Reply:
Also, I think this should be a level 2 module, with a level 1 heading in the master. The parent header could be "Using the MTA IDE plugin" and this module could be "Analyzing your projects".

In its current form, it's hard for the user to see, from the TOC, what this section is about because the actual usage section title does not appear in the TOC.